code/modules/clothing/spacesuits/rig/suits/hacker.dm

```diff
@@ -1,38 +0,0 @@
-/obj/item/weapon/rig/light/hacker
-	name = "cybersuit control frame"
-	suit_type = "cyber"
-	desc = "An advanced suit with cyberwarfare enhancements. Suitable for safely tampering with electronics."
-	icon_state = "hacker_rig"
-
-	armor = list(melee = 25, bullet = 25, laser = 25, energy = 25, bomb = 25, bio = 25, rad = 25)
-
-	airtight = 0
-	seal_delay = 5 //not being vaccum-proof has an upside I guess
-
-	helm_type = /obj/item/clothing/head/lightrig/hacker
-	chest_type = /obj/item/clothing/suit/lightrig/hacker
-	glove_type = /obj/item/clothing/gloves/lightrig/hacker
-	boot_type = /obj/item/clothing/shoes/lightrig/hacker
-
-	max_health = 1500
-
-	initial_modules = list(
-		/obj/item/rig_module/healthbar,
-		/obj/item/rig_module/storage/light,
-		/obj/item/rig_module/power_sink,
-		/obj/item/rig_module/electrowarfare_suite,
-		/obj/item/rig_module/voice,
-		/obj/item/rig_module/hotswap
-		)
-
-/obj/item/clothing/head/lightrig/hacker
-
-/obj/item/clothing/suit/lightrig/hacker
-	siemens_coefficient = 0.2
-
-/obj/item/clothing/shoes/lightrig/hacker
-	siemens_coefficient = 0.2
-	item_flags = ITEM_FLAG_NOSLIP //All the other rigs have magboots anyways, hopefully gives the hacker suit something more going for it.
-
-/obj/item/clothing/gloves/lightrig/hacker
-	siemens_coefficient = 0
```

Comment:
Wasn't this intentionally re-designed by nanako and planned for integration into the patron item list?

Reply:
Different hacker suit.

Bay has one as well.